### DISCUSSION TITLE
azure-pipelines-for-private-aks-clusters/

### DISCUSSION THREAD

**Discussion by giscus** [colindembovsky/colindembovsky.github.io#14] [Giscussions]
# azure-pipelines-for-private-aks-clusters/

Creating private AKS clusters is a good step in hardening your Azure Kubernetes clusters. In this post I walk through the steps you’ll need to follow to enable deployment to private AKS clusters.

https://colinsalmcorner.com/azure-pipelines-for-private-aks-clusters/

**Comment by rohancragg** [comment]
This post is an absolute lifesaver! many thanks

**Comment by ankurpuri1981** [comment]
This is really a wonderful post with all details.

**Comment by jcregg2** [comment]
Thanks for this post, really useful. I have followed and I am getting a problem where the Kubernetes task shows "Could not fetch Kubectl version. Please make sure that the Kubernetes server is up and running." The pipeline doesn;t fail until it times out, but it seems like it can't connect the cluster. I have specified the correct version and I can push from my local using kubectl. Any ideas?
